pixi/backend/api.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+const {dummyApiResponse} = require('./constants.js');
```

Comment:
tip: you can directly require json files (`const dummyResponse = require('file.json')`)